### PR TITLE
Surface level raw download configs added. 

### PR DESCRIPTION
**Pull request by dabhicusp**
By analyzing the existing data of `lnsp` and `z`(Geopotential) variables and cross-referencing it with data availability from the [ECMWF official catalogue](https://apps.ecmwf.int/mars-catalogue/), I have successfully generated surface level configs. This resolves issue #33.

### CODE REVIEW

File: raw/era5_ml_lnsp.cfg [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+# -------------------------------------------|------------|-----------------------|-----------------------------------------------------
+# name                                       | short name | units                 | docs
+# -------------------------------------------|------------|-----------------------|-----------------------------------------------------
+# Logarithm of surface pressure              | lnsp       | Numeric               | https://apps.ecmwf.int/codes/grib/param-db?id=152
+# -------------------------------------------|------------|-----------------------|-----------------------------------------------------
+param=152.128
+
+
+
+
```
Comment (alxmrs) [inline raw/era5_ml_lnsp.cfg:45]: Please remove extra whitespace here.

File: raw/era5_ml_zs.cfg [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+time=00/to/23
+# ------------------------------|------------|----------|--------------------------------------------------
+# name                          | short name | units    | docs
+# ------------------------------|------------|----------|--------------------------------------------------
+# Geopotential                  | z          | m^2 s^-2 | https://apps.ecmwf.int/codes/grib/param-db?id=129
+# ------------------------------|------------|----------|--------------------------------------------------
+param=129.128
+
+
+
```
Comment (alxmrs) [inline raw/era5_ml_zs.cfg:44]: Please remove.

**Review by alxmrs — APPROVED** [review]
One small nit, otherwise LGTM.

File: raw/era5_ml_lnsp.cfg (hunk 1 above)
Comment (dabhicusp) [reply raw/era5_ml_lnsp.cfg]: Removed empty spaces.

**Pull request merged**